MagneticField/VolumeGeometry/interface/MagneticFieldVolume.h

```diff
@@ -1,35 +0,0 @@
-#ifndef MagneticFieldVolume_H
-#define MagneticFieldVolume_H
-
-class MagneticFieldVolume {
```

Comment:
@namapane 
do you happen to know the history of this file?

Reply:
it looks like this was introduced in 2005 d4bdb0cb71d9c69743103e4b5363c8e7c39eac60 and never used since.